docs/docs/reference/metaprogramming/macros-spec.md

```diff
@@ -9,14 +9,14 @@ title: "Macros Spec"
 
 Compared to the [Dotty reference grammar](../../internals/syntax.md)
 there are the following syntax changes:
-
-      SimpleExpr      ::=  ...
-                        |  ‘'’ ‘{’ Block ‘}’
-                        |  ‘'’ ‘[’ Type ‘]’
-                        |  ‘$’ ‘{’ Block ‘}’
-      SimpleType      ::=  ...
-                        |  ‘$’ ‘{’ Block ‘}’
-
+```none
```

Comment:
Do we gain anything by using `none` instead of just leaving it empty? If no I would rather not have it as it adds noise when reading the md files directly...

Reply:
If no language is declared, hljs will try to guess it, which leads to incorrect and annoying highlighting, especially for non-code blocks like ASCII graphs.

Reply:
I've found a better solution: 2102ca1
Leaving it empty now works as expected, ie it disables highlighting.